CMakeLists.txt

```diff
@@ -442,6 +443,26 @@ set (UPM_COMMON_HEADER_DIRS ${CMAKE_HOME_DIRECTORY}/include)
 # Generate a build-only C++ header to add functionality to SWIG'ed modules
 configure_file (${PROJECT_SOURCE_DIR}/cmake/modules/version.hpp.in ${PROJECT_BINARY_DIR}/src/version.hpp @ONLY)
 
+# Cordova binding
+if (BUILDCORDOVABINDING)
+  if (NOT BUILDSWIGJAVA)
+    message(FATAL_ERROR "Cordova bindings needs swig java modules to be built first")
+  endif()
+
+  find_package (Node REQUIRED)
+  find_package (Npm REQUIRED)
+  find_package (Cordova REQUIRED)
+
+  add_custom_target(cordova_binding ALL
```

Comment:
Can this be move to the src-level CMakeLists.txt file?  The other language binding creation is done there.